src/Sulu/Bundle/ContactBundle/Contact/ContactManagerInterface.php

```diff
@@ -21,11 +24,11 @@ interface ContactManagerInterface
     /**
      * adds an address to the entity.
      *
-     * @param string $entity The entity to add the address to
+     * @param ContactAddress $entity The entity to add the address to
      * @param Address $address The address to be added
      * @param bool $isMain Defines if the address is the main Address of the contact
      *
-     * @return array $relation
+     * @return ContactAddress $relation
```

Comment:
Same as above for `AccountManager` this is a `AccountAddress` and for the `ContactManager` this is  `ContactAddress` so this should be `object´  or we should use generics for it.

Reply:
For simple I would go here with `ContactAddress|AccountAddress`